src/coreComponents/fileIO/vtk/VTKPolyDataWriterInterface.cpp

```diff
@@ -379,7 +379,7 @@ void VTKPolyDataWriterInterface::WriteEmbeddedSurfaceElementRegions( real64 time
                                                                      NodeManager const & nodeManager,
                                                                      EdgeManager const & edgeManager ) const
 {
-  elemManager.forElementRegions< EmbeddedSurfaceRegion >( [&]( EmbeddedSurfaceRegion const & er )->void
+  elemManager.forElementRegions< SurfaceElementRegion >( [&]( SurfaceElementRegion const & er )->void
```

Comment:
Do you know if only one `VTKWriter` can be used?

Reply:
Yes, once my other PR (836) gets merged. I have some changes in there that should allow me to have a single vtkWriter!